bundles/persistence/org.openhab.persistence.sitewhere/src/main/java/org/openhab/persistence/sitewhere/internal/SiteWhereConfigurationManager.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.persistence.sitewhere.internal;
+
+import java.io.File;
+
+import org.openhab.config.core.ConfigDispatcher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Manages SiteWhere configuration options.
+ * 
+ * @author Derek
+ */
+public class SiteWhereConfigurationManager {
```

Comment:
If I remember correctly, we initially tried to use openhab.cfg, but had problems where the configuration was not always available when we started the persistence provider. Loading from a well-known file allowed us to get around the issues. We can take another look at using the openhab.cfg to get rid of the proprietary aspects.